src/styles/utils/variables.scss

```
@@ -0,0 +1,7 @@
+$tablet-min-width: 576px;
+$desktop-min-width: 1024px;
+
+$effectDuration: 0.3s;
```

Use one style of writing variables
```suggestion
$effect-duration: 0.3s;
```